CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## 1.26.0 [unreleased]
 
+### Features
+
+1. [#442](https://github.com/influxdata/influxdb-client-js/pull/442): Regenerate APIs from swagger.
+
+### Other
+
+1. [#442](https://github.com/influxdata/influxdb-client-js/pull/442): Rename `InvocableScripts` to `InvokableScripts`.
```

Comment:
This PR mainly contains 2 things:
- regenerates APIs from swagger
- renames `invocableScripts.js` to `invokableScripts.js` + add a few log/doc replacements to `invokable`

The `Other` change does not describe the action taken, there was no `InvocableScripts` before
```suggestion
1. [#442](https://github.com/influxdata/influxdb-client-js/pull/442): Demonstrate invokable scripts in `invokableScripts.js` example.
```